recipes-test/demo-secondary-config/secondary-config.bb

```diff
@@ -0,0 +1,42 @@
+DESCRIPTION = "Sample configuration for an Uptane Secondary"
+LICENSE = "CLOSED"
```

Comment:
Closed? Why not MPL-2.0 like the rest of meta-updater?

Reply:
Don't know, just copy pasted one of the recipes under recipes-test. Will fix it.